### ISSUE TITLE
Dark theme

### ISSUE THREAD

**Issue by johan12345** [opacapp/opacclient#558]
Some users have suggested that the app should offer a dark / night theme.

It should be quite easy to implement this, but I am not sure what is the most up-to-date way to implement this, so we should do some research about this. I found this post from 2016:
https://medium.com/androiddevelopers/appcompat-v23-2-daynight-d10f90c83e94

There will be a new system-wide *dark mode* in Android Q, so we should make sure that it works natively there. But if possible, it should also be available for users on older versions of Android.

**Comment by iNtEgraIR2021** [comment]
This source might be helpful for the implementation:

* [proandroiddev.com/implementing-dark-theme-in-your-android-application-ec2b4fefb6e3](https://proandroiddev.com/implementing-dark-theme-in-your-android-application-ec2b4fefb6e3)

@johan12345 you posted your comment above quite a while ago (about 3 years ago as of July 2021) so where there any acitivities related to this issue in the meantime?

**Comment by johan12345** [comment]
No, sorry, we never got around to implementing this. Development of new features for the app from our side ([rami.io](https://rami.io/)) has been reduced to a minimum over the last few years, as the income from the commercial variants of the app that we offer to libraries only barely covers the ongoing support and maintenance (see also our related [post](https://opac.app/de/support-policy/) from 2019). At the same time, the main focus of the company has shifted to [other](https://pretix.eu/) [products](https://venueless.org/).

I personally have also ended my side job at rami.io earlier this year and moved on to [other projects](https://johan.vonforst.net/), so while I still [occasionally contribute](https://github.com/opacapp/opacclient/pull/640) to the app in my spare time, @raphaelm is the one in charge now. But as far as I know, Open Source contributions are still welcome 🙂
